sample.go

```diff
@@ -789,3 +795,31 @@ func UpdateNicProperties(hmcManager zhmcclient.ZhmcAPI) {
 		}
 	}
 }
+
+func GetEnergyFromLpar(hmcManager zhmcclient.ZhmcAPI) {
+
+	lparID := os.Getenv("PAR_ID")
+	lparURI := "api/logical-partitions/" + lparID
```

Comment:
I noticed this lparURI does not has `/` at beginning, while same variable at L818 starts as `/`, are they all good?

Reply:
yes, but we should keep consistency and i add"/" at the beginning.